.builder/actions/aws_crt_java_test.py

```diff
@@ -11,6 +11,8 @@ def _run_java_tests(self, *extra_args):
             os.remove('log.txt')
 
         profiles = 'continuous-integration'
+        if os.getenv("AWS_GRAAL_VM_CI") is not None:
```

Comment:
utterly trivial/debatable:  GraalVM is always one word, feels weird as two
```suggestion
        if os.getenv("AWS_GRAALVM_CI") is not None:
```